neuro-sdk/src/neuro_sdk/_utils.py

```diff
@@ -27,6 +27,14 @@
 from ._errors import ConfigError
 from ._rewrite import rewrite_module
 
+
+class OrgNameSentinel:
+    pass
```

Comment:
If add a special class for a singleton, I would add `__repr__`.

Reply:
Sorry, I'm not getting your point. What kind of repr do you suggest? When will it be used?

Reply:
`__repr__` method of `OrgNameSentinel` would be good to have but it is not mandatory.